### PR TITLE
[CfgEditor] Add codicon events

### PR DESCRIPTION
**Pull request by llFreetimell**
This commit adds following events for codicon
- `codicon-search` : Open file dialog to select file or directory.
- `codicon-question` : Show help message when mouseover.
And this commit also removes some unnecessary codicons.

ONE-vscode-DCO-1.0-Signed-off-by: Seok NamKoong <seok9311@naver.com>

---

Draft : #1098 

---

#### With this PR

codicon

### CODE REVIEW

**Comment by yunjayh** [comment]
Q) When the file open dialog has been popped up, the file filter is applied (it is `target files`). What's that mean?

**Review by dayo09 — DISMISSED** [review]

**Comment by llFreetimell** [comment]
> Q) When the file open dialog has been popped up, the file filter is applied (it is target files). What's that mean?

If the dialog is for `one-quantize`, `circle` file would be filtered.
If the dialog is for `one-import-tflite`, `tflite` file would be filtered.
`target files` means those `circle` or `tflite` :)
I couldn't think about great word... :(

File: src/CfgEditor/CfgEditorPanel.ts [hunk 1]
```diff
@@ -106,6 +106,22 @@ export class CfgEditorPanel implements vscode.CustomTextEditorProvider {
             vscode.workspace.applyEdit(edit);
           }
           break;
+        case 'getPathByDialog':
+          const dialogOptions = {
+            canSelectMany: false,
+            canSelectFolders: e.isFolder,
+            openLabel: 'Open',
+            filters: {'target files': e.ext, 'all files': ['*']}
```
Comment (yunjayh) [inline src/CfgEditor/CfgEditorPanel.ts:114]: (Optional) How about this kind of approach?

```suggestion
            filters: {`target files(*.${e.ext})`: e.ext, 'all files': ['*']}
```

image

I followed how windows file open dialog shows.

**Review by yunjayh — DISMISSED** [review]

File: src/CfgEditor/CfgEditorPanel.ts (hunk 1 above)
Comment (llFreetimell) [reply src/CfgEditor/CfgEditorPanel.ts:114]: It looks good but key value should be constant I think :(
I will handle it later...!

**Review by yunjayh — APPROVED** [review]

**Review by dayo09 — APPROVED** [review]

**Pull request merged**